app/styles/custom-utilities/_autocomplete.scss

```diff
@@ -0,0 +1,26 @@
+.autocomplete-results{
+  display: block;
+  position: absolute;
+  background-color: white;
+  margin-top:1em;
+  border:1px solid gray;
+
+  li{
+    position: relative;
+    color:black;
+    text-decoration: none;
+
+    div{
+      display: block;
+    }
+    a{
+      a{
+        padding:0;
+      }
+    }
+  }
+}
```

Comment:
Ik ga er vanuit dat deze klasse er is om het op de juiste positie te krijgen en dat dit verschillend is vanaf het prototype?